### ISSUE TITLE
Stuck at "Setting things up" after "unfortunately the process com.android.phone has stopped"

### ISSUE THREAD

**Issue by Aralox** [sravan953/gopro#2]
Hi Sravan, I'm using a Hero 5 black with an arm7 Motorola G4 plus running android 6.0.1. 
I've managed to mess around with the app enough to get it as far as "Setting things up" (I had to reconnect the wifi to the gopro multiple times in different combinations, and play with the official Capture app while doing it), but after an error message pops up ("unfortunately the process com.android.phone has stopped"), the app does not make any progress towards setting up the stream on youtube.
I haven't been able to log in to facebook through the app - after clicking "log in with facebook", a spinner shows up for around 30 seconds, then it disappears without anything happening.

Could you please give me a hand with getting youtube streaming working?
Thanks :)

**Comment by sravan953** [comment]
A `com.android.phone` issue might be happening on dual-SIM devices. What device you running on?

**Comment by Aralox** [comment]
Yeah it is a dual SIM actually, its a Motorola G4 plus. I see you mentioned that this app might not work for dual SIM devices, my bad. I spent a lot of time trying to get the cellular connection to work while the wifi was active, I thought that was the source of the problem, but I guess it must be something else.
